setup.py

```diff
@@ -12,7 +12,7 @@ def read_requirements():
 
 setup(
     name="mani_skill2",
-    version="0.3.2",
+    version="0.3.3",
```

Comment:
should be 0.4.0?

Reply:
I will bump the version when I actually release 0.4.0 to pypi. That commit is where I commit for 0.3.3